test/new/db/archos/linux-x64/dbg_bps

```diff
@@ -1,7 +1,7 @@
 NAME='dbg.dc+ds.swstep'
 FILE=../bins/elf/analysis/elf-nx
 ARGS=-d
-CMDS=<<EXPECT
+CMDS=<<EOF
```

Comment:
why this change? wasn't the whole purpose of using <<EXPECT to avoid having to introduce EOF?

Reply:
the <<EXPECT thing is was just for backward compat for lazines, i want the parser to be simple and be able to move K=V easily upp and down without breaking because the next statement is not a keyword. this was kind of a hack and we must kill it, the asm suite also have a bunch of old syntaxes that im trying to avoid. the v suite is throwing warnings at them